docs/contributing.md

```diff
@@ -116,9 +118,10 @@ These are listed roughly in order of decreasing importance.
    toolchain and use `cargo +nightly fmt`.
 
 3. Your code will be rejected if it cannot be compiled with the minimal
-   supported version of Rust ("MSRV"). This version is listed as
-   `rust-version` in [`Cargo.toml`](../Cargo.toml); it is 1.64 as of this
-   writing.
+   supported version of Rust ("MSRV"). Currently, `jj` follows a rather
+   casual MSRV policy: "The current `rustc` stable version, minus one."
+   This version is always listed in
+   [`rust-toolchain.toml`](../rust-toolchain.toml).
```

Comment:
This should also be moved to that other PR now. The commit message's mention of rust-toolchain should also be (re)moved